QUEST_LV_0100.tsv

```diff
@@ -5248,12 +5248,12 @@ QUEST_LV_0100_20150317_005247	Read the last epitaph in Apati Cliff
 QUEST_LV_0100_20150317_005248	{memo X}It is the last clue prepared by King Zachariel. Read the epitaph in Apati Cliff and get the fifth clue. 
 QUEST_LV_0100_20150317_005249	{memo X}Got the fifth clue. Go back and talk to Rexipher.
 QUEST_LV_0100_20150317_005250	Protect the Mausoleum Stone Lantern
-QUEST_LV_0100_20150317_005251	The guardian statue says the flow of spell in the Mausoleum will be twisted if the stone lanterns are destroyed. Protect the stone lanterns from demons. 
+QUEST_LV_0100_20150317_005251	The Protector Statue says the flow of spell in the Mausoleum will be twisted if the stone lanterns are destroyed. Protect the stone lanterns from demons. 
```

Comment:
The Protector Statue -> The Guardian's Stone Statue
Mausoleum -> Royal Mausoleum